admin-dev/themes/new-theme/js/pages/feature-flag/index.js

```diff
@@ -29,11 +29,33 @@ const {$} = window;
 
 $(() => {
   const $submitButton = $('#submit-btn-feature-flag');
+  $submitButton.prop('disabled', true);
   const $form = $('#feature-flag-form');
+  const $formInputs = $('#feature-flag-form input');
+  const initialState = $form.serialize();
+  const initialFormData = $form.serializeArray();
+
+  $formInputs.change(() => {
+    $submitButton.prop('disabled', initialState === $form.serialize());
+  });
 
   $submitButton.on('click', (event) => {
     event.preventDefault();
 
+    const formData = $form.serializeArray();
+
+    if (initialState === $form.serialize()) {
+      return;
+    }
+
+    let oneFlagIsEnabled = false;
+    for (let i = 0; i < formData.length; i += 1) {
```

Comment:
@matks some eslint fixing and I think we're good!

Reply:
I have to fix eslint after I applied changes from @PierreRambaud that I disagree with 😛  (but I merged them to just get this delivered)

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/24052/commits/d86e32614c11df01c86e0317e5b5959c31d23d93